redwood-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/lazylayout/uiview/UIViewLazyList.kt

```diff
@@ -99,10 +93,6 @@ internal class ViewPortItems : Widget.Children<UIView> {
     return ViewPortItem(placeholders[placeholderIndex], containerSize())
   }
 
-  internal fun itemCount(): Int {
-    return max(itemsBefore - 1, 0) + viewPortList.count() + itemsAfter
```

Comment:
This was a bug. Now that our `itemCount()` calculation is consolidated in `WindowedItemList`, such a discrepancy between Android an iOS shouldn't occur.

Reply:
Ahh I think this might fix an issue I'm seeing on iOS activity feed